### ISSUE TITLE
VBA Blanks removal issue

### ISSUE THREAD

**Issue by 05Sandy** [AllenMattson/VBA#4]
Hello All,

I have written a VBA code to remove blanks in the particular column. But it removing all the blanks and unfortunately its excluding a blank and not removing that blank row. Can you please help me out where i went wrong.

Help me with the code to remove blanks in a specific column.

Thanks for your response

**Comment by abdullah247** [comment]
yes, definitely try looping in reverse from last row to first rather than top to bottom as it will not work if there are two or more consecutive blank rows.
